src/sentry_plugins/vsts/repository_provider.py

```diff
@@ -71,26 +71,58 @@ def create_repository(self, organization, data, actor=None):
     def delete_repository(self, repo, actor=None):
         pass
 
+    def transform_changes(self, patch_set):
+        type_mapping = {
+            'add': 'A',
+            'delete': 'D',
+            'edit': 'M',
+        }
+        file_changes = []
+        # https://docs.microsoft.com/en-us/rest/api/vsts/git/commits/get%20changes#versioncontrolchangetype
+        for change in patch_set:
+            change_type = type_mapping.get(change['changeType'])
+
+            if(change_type and change['item'] and change['item']['gitObjectType'] == 'blob'):
+                file_changes.append({
+                    'path': change['item']['path'],
+                    'type': change_type
+                })
+
+        return file_changes
+
+    def zip_commit_data(self, repo, commit_list, actor):
+        client = self.get_client(actor)
+        n = 0
+        for commit in commit_list:
+            commit.update(
+                {'patch_set': self.transform_changes(
+                    client.get_commit_filechanges(
+                        repo.config['instance'], repo.external_id, commit['commitId'])
+                )})
+
+            n += 1
+            if n > 90:
```

Comment:
small nit, i think it's nice to put something like this in a constant like `MAX_FILE_CHANGES` or something so that when we inevitably are looking back three months from now to see if there is a limit because a customer has asked, it's easy to find

Reply:
true!